docs/proposals/registry/sample-vs-stack.md

```diff
@@ -0,0 +1,208 @@
+# Proposal for sample vs stack in a devfile registry
+
+## Overview
+
+The existing devfile registry focuses on the stack support to provide generic language/framework/runtime, e.g. Node, Java maven, Quarkus, etc.,  support to build and run user applications. These devfiles are called ***stack devfiles***.  There is another kind of devfile in a devfile registry that is tailored for a building and running a specific application. These devfiles are referred to as ___*sample devfiles*___ (example: https://github.com/redhat-developer/devfile-sample).
```

Comment:
For consistency, we should use either `***text***` or `___*text*___` for bold-italics.

Reply:
will fix